test/harmony-regression/notebook_helpers/requirements.txt

```diff
@@ -0,0 +1,11 @@
+requests >= 2.23.0
```

Comment:
Is this completely replaced by conda now?  Can it be deleted?

Reply:
yes